content/1_General/Input_Output.mdx

```diff
@@ -733,9 +780,10 @@ public class Main {
 		int a = Integer.parseInt(st.nextToken());
```

Comment:
Can you move the buffered reader and printwriter in here? it avoids doing all this try catch stuff up above